execution_engine_testing/test_support/src/step_request_builder.rs

```diff
@@ -48,6 +48,13 @@ impl StepRequestBuilder {
         self
     }
 
+    /// Appends the given vector of [`RewardItem`] into `reward_items`.
+    pub fn with_reward_items(mut self, reward_items: Vec<RewardItem>) -> Self {
```

Comment:
```suggestion
    pub fn with_reward_items(mut self, reward_items: impl IntoIterator<Item = RewardItem>) -> Self {
```
This way you could call it with any iterable, and users can use arrays instead of vectors etc.

Reply:
Done in [`bc80b7d` (#2753)](https://github.com/casper-network/casper-node/pull/2753/commits/bc80b7dfcaba9b846c5d7e01aeef6f52e8042867)